decode.rb

```diff
@@ -0,0 +1,34 @@
+def decode_char(str)
+  decoder = {
+    '.-' => 'A', '-...' => 'B',
+    '-.-.' => 'C', '-..' => 'D',
+    '.' => 'E', '..-.' => 'F',
+    '--.' => 'G', '....' => 'H',
+    '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N',
+    '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R',
+    '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V',
+    '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+
+  decoder[str]
+end
+
+def decode_str(str)
+  words_arr = str.split('  ')
+  phrase = ''
+  words_arr.each do |morse_word|
+    morse_word_arr = morse_word.split
+    morse_word_arr.each do |morse_char|
+      phrase += decode_char(morse_char)
+    end
+    phrase += ' '
+  end
+  print phrase
```

Comment:
- [ ] This method should take a **_morse code input sentence_** as a parameter. You should split the morse code input sentence by **_three spaces_**. After that, you should iterate through the sentence and use the method that you will create for splitting through input words to concatenate the sentence. 

#### Hint:

`decode("-- -.-- -. .- -- .")` should return `"MY NAME"`.